### ISSUE TITLE
All the SVG icons need s focusable="false" attribute for IE11

### ISSUE THREAD

**Issue by afercia** [WordPress/twentynineteen#265]
See related issue / PR for Gutenberg

https://github.com/WordPress/gutenberg/issues/1578
https://github.com/WordPress/gutenberg/pull/2590

And following issues 
https://github.com/WordPress/gutenberg/issues/5247
https://github.com/WordPress/gutenberg/issues/9269

In a few words:
IE 11 suffers from a bug where all SVG icons `inside focusable elements` default to `focusable=”true”`. As a consequence, both the wrapper focusable element and the inner SVG get focus and create a tab stop. To navigate with the keyboard through a button or a link which contain an SVG icon, there's the need to press Tab twice, which is something that should be definitely avoided.

As far as I see this would be a very quick change in `TwentyNineteen_SVG_Icons::get_svg` which already adds `aria-hidden="true" role="img"`.

**Comment by kjellr** [comment]
@afercia This should be handled via #272. Would you mind testing that out to verify that this change removes that tab stop?

Thanks!

**Comment by afercia** [comment]
@kjellr it does fix it thanks! 🎉

**Comment by kjellr** [comment]
Awesome, thank you!
